client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -7,15 +7,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
 
-const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const EditorRevisionsListHeader = ( {
+	numRevisions,
+	translate,
+	selectNextRevision,
+	selectPreviousRevision,
+} ) => {
 	return (
-		<div className="editor-revisions-list__header">
-			{ !! numRevisions &&
-				translate( '%(revisions)d revision', '%(revisions)d revisions', {
-					count: numRevisions,
-					args: { revisions: numRevisions },
-				} ) }
+		<div>
+			<div className="editor-revisions-list__header">
+				{ !! numRevisions &&
+					translate( '%(revisions)d revision', '%(revisions)d revisions', {
+						count: numRevisions,
+						args: { revisions: numRevisions },
+					} ) }
+			</div>
+			<div className="editor-revisions-list__navigation">
+				<Button
+					compact
+					borderless
+					className="editor-revisions-list__prev-button"
+					type="button"
+					onClick={ selectPreviousRevision }
+				>
+					<Gridicon icon="chevron-down" />
+				</Button>
+				<Button
+					compact
+					borderless
+					className="editor-revisions-list__next-button"
+					type="button"
+					onClick={ selectNextRevision }
+				>
+					<Gridicon icon="chevron-up" />
```

Comment:
At smaller viewports, the list changes to scroll horizontally.  We need to either hide the `editor-revisions-list__navigation` class at small viewports or dynamically render the chevrons -- probably in CSS.

What do you think?

Reply:
Whoops, I missed that! When I saw no "History" link at the 660px breakpoint I assumed  the revisions list was hidden on small layouts. That does make sense - I'll sort it out.